src/experiments/smallButtonDesign/eligibility.js

```diff
@@ -0,0 +1,8 @@
+/* @flow */
+
+import type { ButtonProps } from '../../types';
+
+
+export const eligible = ({ props = {} } : {| props : ButtonProps |}) : boolean => {
+    return !props.style?.tagline || false;
```

Comment:
@ssono Small button design eligibility is based on tagline?

Reply:
Yes we only want to apply button designs when we know that there will not be a tagline